Cargo.toml

```
@@ -282,4 +284,21 @@ opt-level = 3
 
 [patch.crates-io]
 # Temporary patch until our PRs are merged.
-tar = { git = "https://github.com/qdrant/tar-rs", branch="main" }
+tar = { git = "https://github.com/qdrant/tar-rs", branch = "main" }
+
+[package.metadata.deb]
+maintainer = "Qdrant Team <team@qdrant.com>"
+depends = "$auto"
+features = ["deb"]
+license-file = "LICENSE"
+section = "database"
+priority = "optional"
+extended-description = """\
+Qdrant is a high-performance vector search engine. It is designed to index and search large collections of high-dimensional vectors.\
+"""
+assets = [
+    ["target/release/qdrant", "usr/bin/", "755"],
+    ["README.md", "usr/share/doc/qdrant/README", "644"],
+    ["static/*/*", "/etc/qdrant/static", "644"],
+    ["config/deb.yaml", "etc/qdrant/config.yaml", "644"],
```

Only one of the lines has the `/` prefix. Is that intended?

---

All examples are actually using path without `/`, so I think it is safe to remove

---

It should be a relative path to properly resolve it when installing the package.

```suggestion
    ["target/release/qdrant", "usr/bin/", "755"],
    ["README.md", "usr/share/doc/qdrant/README", "644"],
    ["static/*/*", "etc/qdrant/static", "644"],
    ["config/deb.yaml", "etc/qdrant/config.yaml", "644"],
```